tests/Fixer/ClassNotation/OrderedClassElementsFixerTest.php

```diff
@@ -449,18 +449,21 @@ public function testLegacyFixWithConfiguration(array $configuration, $expected)
         static $input = <<<'EOT'
 <?php
 
-class Foo
+abstract class Foo
```

Comment:
You updated existing tests which looks risky to me: maybe they covered specific cases they don't cover anymore. It would be better to add specific tests for the new feature you introduced.

Reply:
Cool, I did that just now. I did have to rework `provideConfigurationCases` to also send the input like `provideSortingConfigurationCases` does. I added the modified test cases as new entries in both these methods.